core/source/Set.mint

```diff
@@ -120,4 +120,127 @@ module Set {
   fun toArray (set : Set(item)) : Array(item) {
     `#{set}`
   }
+
+  /*
+  Applies the function against an accumulator and each element in the set (in
+    arbitrary order, according to the set's underlying array representation) to
```

Comment:
I think the order is not arbitrary according to this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

> The reduce() method of [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) instances executes a user-supplied "reducer" callback function on each element of the array, in order, passing in the return value from the calculation on the preceding element.

Reply:
Great, thanks!